examples/bicycle_muscle_steer.py

```diff
@@ -749,15 +749,37 @@ def __init__(self, *args, **kwargs):
 print('Initial speeds')
 print(u_vals)
 
+
+def eval_e(roll_rate):
+    """Specify muscle excitation as a function of time.
+
+    We want the right bicep and left tricep to excite when the roll rate is
+    positive, and the left bicep and right tricep to excite when the roll
+    rate is negative.
+
+    """
+    max_roll_rate = 0.2
+    if roll_rate > 0.0:
+        normalized_roll_rate = roll_rate / max_roll_rate
+        e_bicep_r = normalized_roll_rate if normalized_roll_rate < 0.99 else 0.99
```

Comment:
I think that it is more efficient to use `e_bicep_r = min((normalized_roll_rate, 0.99))`

Reply:
I agree, that much cleaner. Jason and I were rushing to try and get this running before lunch so there are a few horrors like this in this script.